Alignment/HIPAlignmentAlgorithm/validation.ini

```diff
@@ -0,0 +1,71 @@
+[general]
+# please adjust
+jobmode = lxBatch, -q cmscaf1nd
+datadir = /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/results/$USER/20160418_CRUZET16_OfflineValidation/
+#datadir=/afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN2/HIP/public/AlignmentCamp2016/CMSSW_8_0_3/src/Alignment/HIPAlignmentAlgorithm/
+
+[compare:Tracker]
+levels= "DetUnit", "Tracker"
+dbOutput = false
+#jobmode = interactive
+
+[offline:validation_CRUZET16_PromptReco]
+jobmode = lxBatch, -q cmscaf1nd
+maxevents = -1
+dataset = Dataset_Commissioning2016-TkAlCosmics0T-PromptRecov1_ALCARECO_CRUZET
+trackcollection = ALCARECOTkAlCosmicsCTF0T
+offlineModuleLevelHistsTransient = False
+offlineModuleLevelProfiles = True
+DMROptions = plain 
+DMRMinimum = 5
+DMRMethod = median
+
+[offline:validation_CRUZET16_Express]
+jobmode = lxBatch, -q cmscaf1nd
+maxevents = -1
+dataset = Dataset_Commissioning2016-TkAlCosmics0T-StreamExpress_ALCARECO_CRUZET
+trackcollection = ALCARECOTkAlCosmicsCTF0T
+offlineModuleLevelHistsTransient = False
+offlineModuleLevelProfiles = True
+DMROptions = plain
+DMRMinimum = 5
+DMRMethod = median
+
+[alignment:GT_prompt]
+globaltag = 80X_dataRun2_Prompt_v5
+color = 1
+style = 2
+condition TrackerAlignmentRcd =frontier://FrontierProd/CMS_CONDITIONS,TrackerAlignment_prompt
+
+[alignment:GT_iov9]
+globaltag = 80X_dataRun2_Prompt_Candidate_2016_04_18_10_23_57
+color = 1
+style = 1
+condition TrackerAlignmentRcd =frontier://FrontierProd/CMS_CONDITIONS,TrackerAlignment_2015-EOY_IOV09
+
+[alignment:mp2002_BPIX]
+globaltag = 80X_dataRun2_Prompt_Candidate_2016_04_18_10_23_57
+condition TrackerAlignmentRcd = sqlite_file:/afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/MP/MPproduction/mp2002/jobData/jobm/alignments_MP.db, Alignments
+condition TrackerAlignmentErrorExtendedRcd = zeroAPE
+color = 2
+style = 1
+
+[alignment:mp2004_ALL]
+globaltag = 80X_dataRun2_Prompt_Candidate_2016_04_18_10_23_57
+condition TrackerAlignmentRcd = sqlite_file:/afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/MP/MPproduction/mp2004/jobData/jobm/alignments_MP.db, Alignments
+condition TrackerAlignmentErrorExtendedRcd = zeroAPE
+color = 3
+style = 1
+
+[alignment:hp1614_Barrel]
+globaltag = 80X_dataRun2_Prompt_Candidate_2016_04_18_10_23_57
+color = 4
+style = 2
+condition TrackerAlignmentRcd = sqlite_file:/afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN2/HIP/public/AlignmentCamp2016/CMSSW_8_0_3/src/Alignment/HIPAlignmentAlgorithm/hp1614/alignments_iter20.db,Alignments
+
+[validation]
+#DMRs
+offline validation_CRUZET16_PromptReco: GT_iov9
+offline validation_CRUZET16_PromptReco: hp1614_Barrel
+#Geometry Comparisons
+compare Tracker: hp1614_Barrel 262922, GT_iov9 262922
```

Comment:
Does this file really belong into `Alignment/HIPAlignmentAlgorithm`? It's a validation configuration and will be outdated quickly, anyway.